### DISCUSSION TITLE
GitHub Education Transfer Due to Forgotten Account

### DISCUSSION THREAD

**Discussion by MrCLorenz** [community/community#130788] [GitHub Education]
### Hi everyone,

Hello,

I would like to ask if it is possible to transfer my old GitHub Education account because I forgot my account credentials along with the recovery codes. I created a new account and unlinked the emails from my old account in order to regain GitHub Education access.

Thank you!

**Comment by davevad93** [comment]
Hi @MrCLorenz ,

It's not possible to transfer student benefits from one account to another, you could merge two accounts (see [Merging multiple personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts)) but the account with the benefits would be the main one and considering that you lost your credentials, this wouldn't be a valid solution.
